app/Http/Controllers/NotificationController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Models\Notification;
+use Illuminate\Support\Facades\Auth;
+
+class NotificationController extends Controller
+{
+    public function index() 
+    {
+      
+        $login_user = auth::user();
+        $notification_read = $login_user->unread_notifications()->get();
```

Comment:
getした結果は配列（複数）になるので、変数名も複数形にしましょう
```suggestion
        $notification_reads = $login_user->unread_notifications()->get();
```